### PR TITLE
Adds support for Pulse devices >= Release 9.1R7

### PR DESCRIPTION
**Pull request by achard**
Pulse Connect Secure updated the sysobjectid in a recent release.
This PR updates the pulse.yaml definition to include the latest sysobjectid 

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by achard** [comment]
This addresses the issue mentioned here
https://community.librenms.org/t/pulse-secure-psa-3000-9-1r8/13095

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/pulse-secure-psa-3000-9-1r8/13095/3

File: includes/definitions/pulse.yaml [hunk 1]
```diff
@@ -10,3 +10,4 @@ discovery:
     -
         sysObjectID:
             - .1.3.6.1.4.1.12532.
+            - .1.3.6.1.4.1.8072.
```
Comment (Jellyfrog) [inline includes/definitions/pulse.yaml:13]: This would match a lot of devices, is doesnt belong to pulse secure.

Comment (Jellyfrog) [inline includes/definitions/pulse.yaml:13]: This would match a lot of devices, is doesnt belong to pulse secure.

**Review by Jellyfrog — CHANGES_REQUESTED** [review]

File: includes/definitions/pulse.yaml (hunk 1 above)
Comment (achard) [reply includes/definitions/pulse.yaml:13]: Hmm, yes I see.
This isn't as simple as I had hoped, I will cancel this PR and see if I can work out something else.

**Pull request closed without merging**